### PR TITLE
Document errors and warnings about external changes/services

### PR DESCRIPTION
**Pull request by rchodava**

### CODE REVIEW

**Review by razor-x — APPROVED** [review]

File: docs/api-clients/access-codes/README.md [hunk 1]
```diff
@@ -50,10 +48,11 @@ Seam recommends adding error handling logic to you application for each generic
...
+| failed_to_set_on_device        | An error occurred when we tried to set the access code on the device. We will continue to try and set the code on the device in case the error was temporary.                                                                                                                                                            |
+| failed_to_remove_from_device   | An error occurred when we tried to remove the access code from the device. We will continue to try and remove the code from the device in case the error was temporary.                                                                                                                                                  |
+| code_modified_external_to_seam | We detected that the code was modified or removed externally after Seam successfully set it on the device. We will attempt to set the code on the device again. Please advise the user that Seam will re-set codes they modify or remove manually, and to review other lock integrations that may be modifying the codes |
```
Comment (dawnho) [inline docs/api-clients/access-codes/README.md:55]:
```suggestion
| code_modified_external_to_seam | We detected that the code was modified or removed externally after Seam successfully set it on the device. We will attempt to set the code on the device again. Please advise the user that Seam will re-set codes they modify or remove manually, and to review other lock integrations that may be modifying the codes. |
```

File: docs/api-clients/access-codes/README.md [hunk 2]
```diff
@@ -68,17 +67,17 @@ Treat these errors as Specific Errors.
...
+| august_lock_missing_keypad                       | August lock is missing a keypad                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| august_lock_temporarily_offline                  | August lock was temporarily offline while setting/removing a code, the code should be set/remove when it comes back online                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (dawnho) [inline docs/api-clients/access-codes/README.md:76]:
```suggestion
| august_lock_bad_access_code_format               | The access code format is not correct.                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
| august_lock_missing_keypad                       | August lock is missing a keypad.                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
| august_lock_temporarily_offline                  | August lock was temporarily offline while setting/removing a code, the code should be set/remove when it comes back online.                                                                                                                                                                                                                                                                                                                                                            |
```

File: docs/api-clients/devices/README.md [hunk 3]
```diff
@@ -86,11 +86,11 @@ Seam recommends adding error handling logic to you application for each generic
...
-| device_removed                    | Device has been removed from the Connected Account. Seam can no longer sync with this device.                                                                                                                                                                |
-| hub_disconnected                  | The hub that the device is connected to is offline. Seam is unable to sync updates to this device.                                                                                                                                                           |
+| Error Type          | Description                                                                                        |
+| ------------------- | -------------------------------------------------------------------------------------------------- |
+| device_disconnected | Device is disconnected                                                                             |
```
Comment (dawnho) [inline docs/api-clients/devices/README.md:91]:
```suggestion
| device_disconnected | Device is disconnected.                                                                             |
```

**Review by dawnho — APPROVED** [review]
Just some nits

**Comment by razor-x** [comment]
Are these typos in the actual errors? We may need to update the backend error messages to match.

**Comment by dawnho** [comment]
Yeah, they are

**Review by seveibar — APPROVED** [review]

**Pull request merged**